packages/contracts/src/dollar/libraries/LibUbiquityPool.sol

```diff
@@ -87,6 +89,15 @@ library LibUbiquityPool {
         bool[] isMintPaused;
         // whether redeeming is paused for a particular collateral index
         bool[] isRedeemPaused;
+        //====================================
+        // Governance token pricing related
+        //====================================
+        // chainlink price feed for ETH/USD pair
+        address ethUsdPriceFeedAddress;
+        // threshold in seconds when chainlink's ETH/USD price feed answer should be considered stale
+        uint256 ethUsdPriceFeedStalenessThreshold;
```

Comment:
I do not see `ethUsdPriceFeedStalenessThreshold` being used, could you please also add a test for this? Perhaps I am to tired today but I just see setters/ getters only.

Reply:
This PR is a draft